### ISSUE TITLE
Can we Upgrade the old project version to the latest version 0.69 without adopting new architecture?

### ISSUE THREAD

**Issue by robyhuzwandar** [react-native-community/upgrade-support#200]
I have an old project that uses version "react-native": "0.66.3", and I want to upgrade to 0.69 but I don't want to use new architecture to avoid breaking changes on my app.

**Comment by grit96** [comment]
I am also interested in knowing this. I attempted upgrading from 0.68.2 -> 0.69.1 without using the new architecture and encountered lots of build issues so shelved it for another day.

**Comment by Adnan-Bacic** [comment]
im in the middle of updating a project from 0.68 to 0.69 and i have not enabled new architechure. so it should be possible.

**Comment by grit96** [comment]
> im in the middle of updating a project from 0.68 to 0.69 and i have not enabled new architecture. so it should be possible.

Have you got both Android and iOS building on RN 0.69?

**Comment by Adnan-Bacic** [comment]
> > im in the middle of updating a project from 0.68 to 0.69 and i have not enabled new architecture. so it should be possible.
> 
> Have you got both Android and iOS building on RN 0.69?

Yes. Didnt enable new architecture for either platform and i can run both.

**Comment by akshay030123** [comment]
Can anyone help me that how to upgrade my project?
version i am using : 0.61.5 
and want to switch to version above 0.65
i have tried npx react-native upgrade but nothing happens
